### PR TITLE
Updatereadme

### PR DESCRIPTION
**Pull request by hmunfru**
@jframos

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,22 +1,234 @@
-FI-WARE SDC 
+FI-WARE SDC
```
Comment (jframos) [inline README.rst]: ==================
FIWARE SDC | Saggita
==================

File: README.rst [hunk 2]
```diff
@@ -1,22 +1,234 @@
...
 ============
+
 | |Build Status| |Coverage Status| |help stackoverflow|
 
-Software Deployment and Configuration
--------------------------------------
+Introduction
```
Comment (jframos) [inline README.rst]: This sub-section should be removed and the content of this one should be under the main section (above).

File: README.rst [hunk 3]
```diff
@@ -1,22 +1,234 @@
...
+
+
+Installation
+------------
+
+Using FIWARE package repository (recommended)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Refer to the documentation of your Linux distribution to set up the URL of the
+repository where FIWARE packages are available (and update cache, if needed):
```
Comment (jframos) [inline README.rst]: reStructuredText format. Use ``::`` for block of code. Check all issues about that on this document.

File: README.rst [hunk 4]
```diff
@@ -1,22 +1,234 @@
...
+
+In order to check the status of the service, use the following command
+(no special privileges required):
+
+::
+
+    $ service fiware-sdc status
+
+
+API Overview
```
Comment (jframos) [inline README.rst]: You should add here an overview of main requests to the SDC API and not only a link to the whole API documentation::

        API Overview of the main data flow
              - It is a tutorial, not a reference. It does not need to be exhaustive, it needs to guide the user
              - Provide always curl examples for this section

File: README.rst [hunk 5]
```diff
@@ -1,22 +1,234 @@
...
+
+
+Testing
+=======
+
+Unit tests
+----------
+
+The ``test`` target for each module in the SDC is used for running the unit tests in both components of
+SDC GE:
```
Comment (jframos) [inline README.rst]: Add the code to run unittests. ``mvn test``?

File: README.rst [hunk 6]
```diff
@@ -1,22 +1,234 @@
...
+Then, to stop the service, run::
+
+    $ service fiware-sdc stop
+
+We can also force a service restart::
+
+    $ service fiware-sdc restart
+
+
+Configuration file
```
Comment (jframos) [inline README.rst]: I think you should show here how to configure the ``sdc.xml`` file in order to setup the database connection.

**Comment by jframos** [comment]
You should describe in the **Installation** section how to install SDC on Centos using the automated script located in ``scripts/bootstrap/centos.sh``

File: README.rst (hunk 2 above)
Comment (hmunfru) [inline README.rst]: According to the example, it should be there

File: README.rst (hunk 6 above)
Comment (hmunfru) [inline README.rst]: Done!

File: README.rst (hunk 4 above)
Comment (hmunfru) [inline README.rst]: Done!

**Comment by jmmovilla** [comment]
LGTM

**Pull request merged**